src/main/scala/com/cognite/sdk/scala/v1/Client.scala

```diff
@@ -83,20 +83,18 @@ final case class RequestSession[F[_]: Monad: Trace](
   ): F[Response[R]] =
     r(emptyRequest.readTimeout(90.seconds)).send(sttpBackend)
 
-  private val sttpRequest = {
-    val baseRequest = basicRequest
+  private val sttpRequest =
+    basicRequest
       .followRedirects(false)
       .header("x-cdp-sdk", s"CogniteScalaSDK:${BuildInfo.version}")
       .header("x-cdp-app", applicationName)
       .readTimeout(90.seconds)
-    (clientTag, cdfVersion) match {
-      case (Some(tag), Some(ver)) =>
-        baseRequest.header("x-cdp-clienttag", tag).header("cdf-version", ver)
-      case (Some(tag), None) => baseRequest.header("x-cdp-clienttag", tag)
-      case (None, Some(ver)) => baseRequest.header("cdf-version", ver)
-      case (None, None) => baseRequest
-    }
-  }
+      .headers(
+        Seq(
+          clientTag.map(Header("x-cdp-clienttag", _)),
```

Comment:
If the header value is None does that mean the http library skips adding the header?

Reply:
.flatMap(Option::toList) takes care of that by converting to list of option-unwrapped values, worst case we produce no headers and pass via : _* varargs thing